### ISSUE TITLE
PPC/s390 (be/native):  testcase d8-worker-sharedarraybuffer.js failing

### ISSUE THREAD

**Issue by jbajwa** [ibmruntimes/v8#10]
```
> out/ppc64.release/d8 --test --random-seed=-633966520 --nohard-abort --nodead-code-elimination --nofold-constants --harmony-sharedarraybuffer test/mjsunit/mjsunit.js test/mjsunit/d8-worker-sharedarraybuffer.js

test/mjsunit/d8-worker-sharedarraybuffer.js:63: TypeError: [object Uint32Array] is not an integer shared typed array.
    while ((ta0 = Atomics.load(ta, 0)) == 0) {}
                          ^
TypeError: [object Uint32Array] is not an integer shared typed array.
    at Atomics.load (<anonymous>)
    at TestTransfer (test/mjsunit/d8-worker-sharedarraybuffer.js:63:27)
    attest/mjsunit/d8-worker-sharedarraybuffer.js:70:5
```

**Comment by jbajwa** [comment]
Investigating this issue, the failure started after this CL https://chromiumcodereview.appspot.com/2436893003

**Comment by thinkingdust** [comment]
I think it's solved here: fb7841b314e73e665c3c13a24ca8d96eec553ac3

**Comment by jbajwa** [comment]
yup, this issue is fixed.
